md/database-configuration-for-business-data.md

```diff
@@ -24,7 +24,35 @@ Configure the database to use the UTF-8 character set.
 
 In your RDBMS, make sure there is a user defined with privileges to create tables and query them.
 
-Edit [`bonita-tenant-community-custom.properties`](BonitaBPM_platform_setup.md) for each tenant and set the `bdm.db.vendor` property to indicate the RDBMS vendor.
+## Data sources
+
+
+For business data objects, you need to configure two data sources: `BusinessDataDS` and `NotManagedBizDataDS`. Two data sources are required because some RDBMS do not support use of transactions for table creation.
+The data sources for business data objects are independent of the Bonita BPM Engine data sources (`bonitaDS` and `bonitaSequenceManagerDS`).
+
+::: info
+If you have multiple tenants that use business data objects, you need to create a `BusinessDataDS` and a `NotManagedBizDataDS` for each tenant.
+:::
+The following sections show how to configure the data sources for [JBoss](#ds_jboss) and [Tomcat](#ds_tomcat). 
+There is also an [example of how to configure data sources for Weblogic](red-hat-oracle-jvm-weblogic-oracle.md).
+
+After creation of a new Tenant, use the Setup tool to pull the configuration. See [Bonita BPM plaform setup tool](BonitaBPM_platform_setup.md) for more information about platform configuration.
```

Comment:
we usually try to gather contents of same topic in a single page to ease support of these topics.  
I think we should leave the description of how to do a pull/push inside the [Bonita BPM plaform setup tool](../../blob/7.3/md/BonitaBPM_platform_setup.md) page. If we do so, we also should change the sentence to something like: 
`After creation of a new Tenant, use the [Bonita BPM plaform setup tool](BonitaBPM_platform_setup.md) to update the configuration of the newly created tenant.`